.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchTimePeriodParserConfiguration.cs

```diff
@@ -91,6 +91,10 @@ public bool GetMatchedTimexRange(string text, out string timex, out int beginHou
             {
                 timeOfDay = Constants.BusinessHour;
             }
+            else if (DateTimeDefinitions.BusinessHourSplitStrings2.All(o => trimmedText.Contains(o)))
+            {
+                timeOfDay = Constants.BusinessHour;

```

Comment:
Why is this a separate regex from the previous?

Reply:
BusinessHourSplitStrings2 refers to a different expression for "business hour" (like "office hour"), but the if condition checks that all tokens in BusinessHourSplitStrings are contained in trimmedText. 
So if we put both "business hour" and "office hour" inside BusinessHourSplitStrings, the if condition would be satisfied only if trimmedText contains both expressions.